browser/brave_app_controller_mac.h

```diff
@@ -6,14 +6,24 @@
 #ifndef BRAVE_BROWSER_BRAVE_APP_CONTROLLER_MAC_H_
 #define BRAVE_BROWSER_BRAVE_APP_CONTROLLER_MAC_H_
 
+#import <Cocoa/Cocoa.h>
+
 #import "chrome/browser/app_controller_mac.h"
 
 // Manages logic to switch hotkey between copy and copy clean link item.
-@interface BraveAppController : AppController {
-  NSMenuItem* _copyMenuItem;
-  NSMenuItem* _copyCleanLinkMenuItem;
-}
+@interface BraveAppController : AppController
+
+@property(readonly, nonatomic, class) BraveAppController* sharedController;
+
+@property(nonatomic, assign) NSMenuItem* copyMenuItem;
+@property(nonatomic, assign) NSMenuItem* copyCleanLinkMenuItem;
```

Comment:
Is there a reason these are explicitly marked with `assign` and not `strong`? Under an ARC environment we don't really use `assign` for anything except primitives

Reply:
I think @spylogsster is out this week. @sangwoo108 and @simonhong could you, please, chime in.

Reply:
Yup, I'm on it.

Reply:
@sangwoo108 while you're around here, https://github.com/brave/brave-core/pull/18808/files#diff-99de7c0a319466aaa43c894332871ab5eebd0dc842948d103bb41be688bf4d12R139-R141 could be deleted also :)